hikari/impl/rest.py

```diff
@@ -764,6 +764,22 @@ async def edit_channel(
         response = typing.cast(data_binding.JSONObject, raw_response)
         return self._entity_factory.deserialize_channel(response)
 
+    async def follow_channel(
+        self,
+        origin_channel: snowflakes.SnowflakeishOr[channels.GuildNewsChannel],
+        target_channel: snowflakes.SnowflakeishOr[channels.GuildChannel],
```

Comment:
Could we call these `subscribe_to` and `crosspost_to`, just because `target_channel` could refer to the target to subscribe to, or the target to crosspost to, and the `origin` could be the origin of the announcements or the origin of the crossposts that are made.

Reply:
could call it `news_channel` instead of `subscribe_to` but that naming scheme is more consistent

Reply:
ok